### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by ineiti**
Based on PR from @kc1212 

Added linting to the files, so that Travis passes.

### CODE REVIEW

File: go/dist/vec.go [hunk 1]
```diff
@@ -4,14 +4,14 @@ package dist
 type vec []int
 
 // Return a copy of this vector
-func (x vec) copy() vec {
-	return append(vec{}, x...)
+func (v vec) copy() vec {
+	return append(vec{}, v...)
 }
 
 // Return true if vector timestamp x is causally before or equal to y.
```
Comment (nkcr) [inline go/dist/vec.go]:
```suggestion
// Return true if vector timestamp v is causally before or equal to y.
```

File: go/dist/vec.go [hunk 2]
```diff
@@ -20,12 +20,12 @@ func (x vec) le(y vec) bool {
 
 // Set z to the elementwise maximum of vectors x and y.
```
Comment (nkcr) [inline go/dist/vec.go]:
```suggestion
// Set v to the elementwise maximum of vectors x and y.
```

File: go/dist/vec.go [hunk 3]
```diff
@@ -20,12 +20,12 @@ func (x vec) le(y vec) bool {
...
 // Inputs x and/or y can be the same as target z.
```
Comment (nkcr) [inline go/dist/vec.go]:
```suggestion
// Inputs x and/or y can be the same as target v.
```

File: go/dist/dist_test.go [hunk 4]
```diff
@@ -175,8 +178,7 @@ func testExec(t *testing.T, threshold, nnodes int) {
 }
 
 // Exec a child as a separate process.
-func testExecChild(t *testing.T, conf *testConfig, ctx context.Context,
-	grp *sync.WaitGroup) (io.Writer, io.Reader) {
+func testExecChild(ctx context.Context, conf *testConfig, t *testing.T, grp *sync.WaitGroup) (io.Writer, io.Reader) {
```
Comment (nkcr) [inline go/dist/dist_test.go]: I would respect the nice formatting of this file and add a proper return here

File: go/dist/qsc.go [hunk 5]
```diff
@@ -1,6 +1,6 @@
 package dist
 
-// Witnessed QSC requires three TLC time-steps per consensus round.
+// RoundSteps is three because the witnessed QSC requires three TLC time-steps per consensus round.
```
Comment (nkcr) [inline go/dist/qsc.go]: line-width formatting

File: go/model/tlc.go [hunk 6]
```diff
@@ -37,7 +37,7 @@ func (n *Node) Advance() {
 	n.broadcastTLC() // broadcast our raw proposal
 }
 
-// The client or network layer calls this on receipt of a Message from a peer.
+// Receive is called by the client or network layer on receipt of a Message from a peer.
```
Comment (nkcr) [inline go/model/tlc.go]: line-width formatting

File: go/dist/dist_test.go (hunk 4 above)
Comment (ineiti) [reply go/dist/dist_test.go]: I don't understand what you would like to change here (besides the line-width).

**Pull request merged**